app-client/pom.xml

```diff
@@ -62,8 +60,7 @@
         <repository>
             <id>jboss-public-maven-repository</id>
             <name>JBoss Public Maven Repository</name>
-            <url>${maven.jboss.public.repository.url}</url>
-            <layout>default</layout>
+            <url>https://repository.jboss.org/nexus/content/groups/public/</url>
```

Comment:
Can the `<repositories>` be moved to the parent too?

Reply:
@jmesnil you need the repos to at least retrieve the parent, which could be worked around by mirroring the parent to Maven Central. Moving there the repos also goes against the idea that each Quickstart should be as "complete" as possible, hiding in the parent only the stuff (profiles, plugins, etc) that are used for our project specifics (asciidoc, checkstyle, etc), since users would not use this parent.

Reply:
Please note that (when not using the parent) there are environments where we can't workaround and the user apps will need the repos, such as CodeReady Workspaces.